opm/output/data/Wells.hpp

```diff
@@ -120,13 +122,26 @@ namespace Opm {
         void read(MessageBufferType& buffer);
     };
 
+    struct Segment {
+        Rates rates;
+        double pressure;
+        std::size_t segNumber;
```

Comment:
I assume this value be used as value and/or index in various `?SEG` keywords - then it will eventually be `int`; I would suggest just using `int` all the way.

Reply:
Just index, and really only as an index into `std::vector<T>`s.  I think `std::size_t` is appropriate here.